### PR TITLE
chore(ci): docker

### PR DESCRIPTION
**Pull request by luizm**
Configure to create a docker image and push it to the GCR registry, 

I preferred to keep our modification in only one file, in the case named "build.sh"

### CODE REVIEW

**Comment by drewdahlke** [comment]
TODO: bake the resource group config into the container https://github.com/totvslabs/infra/blob/56e338f933f92985c793f7d190020acb13f74e48/scripts/dataproc/presto.sh#L217

**Comment by luizm** [comment]
> TODO: bake the resource group config into the container

I was thinking to create a configmap and mount it as a file, we made a similar to other config files.

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM openjdk:8u151-jdk-alpine3.7
```
Comment (cleberwarmling) [inline docker/Dockerfile]: Any specific reason to use this tag? It's pretty outdated. I think that `openjdk:8u272-jdk-slim` could be an alternative.

File: .buildkite/build.sh [hunk 2]
```diff
@@ -1,10 +1,19 @@
-#!/bin/sh
+#!/bin/bash
 set -e
+image="totvslabs/presto"
 
-echo "~~~ build"
+# echo "~~~ build"
```
Comment (cleberwarmling) [inline .buildkite/build.sh]: 💅 This echo makes "text blocks" on Buildkite... Maybe keeping it is better.

**Review by cleberwarmling — APPROVED** [review]

File: .buildkite/build.sh (hunk 2 above)
Comment (luizm) [reply .buildkite/build.sh]: Typo.. fixing

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    adduser -D presto -u 1000 -G presto && \
+    mkdir -p /usr/lib/presto && \
+    chown -R "presto:presto" /usr/lib/presto
+
+ENV PYTHONUNBUFFERED=1
+RUN echo "**** install Python ****" && \
+    apk add --no-cache python3 && \
+    if [ ! -e /usr/bin/python ]; then ln -sf python3 /usr/bin/python ; fi && \
+    \
+    echo "**** install pip ****" && \
```
Comment (caarlos0) [inline docker/Dockerfile]: i would remove these echos... 💅

**Review by caarlos0 — APPROVED** [review]

**Review by caarlos0 — APPROVED** [review]

**Pull request merged**